docs/changelog/0.5.0.md

```diff
@@ -17,6 +17,7 @@ Check out the documentation at
 * **Not backward compatible with previous releases**,
 * Removed `java.net.URL` and `java.net.URI.toURL` related stubs we don't plan to implement.
 * Various changes to the build system. See "Build Integrator features" below.
+* Introduce new nativeMode `release-size` optimizes for output binary size
```

Comment:
I suggest something like "optimizes for reduced output binary size"

Reply:
thanks, updated